BUILD

```diff
@@ -0,0 +1,10 @@
+cc_library(
+  name = "linenoise-cpp",
```

Comment:
`linenoise.cpp` cannot be used, since it presumably conflicts with the cpp file.